packages/core-mobile/app/new/features/portfolio/assets/hooks/useAssetsFilterAndSort.ts

```diff
@@ -108,7 +109,8 @@ export const useAssetsFilterAndSort = (): {
 
   const filteredAndSorted = useMemo(() => {
     const filtered = getFiltered()
-    return getSorted(filtered)
+    const sorted = getSorted(filtered)
+    return sortedTokensWithBalance(sorted)
```

Comment:
should we just do sortedTokensWithBalance here since it will override the output of getSorted?